kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/BrokersResource.java

```diff
@@ -89,14 +89,6 @@ public void getBroker(
     AsyncResponses.asyncResume(asyncResponse, response);
   }
 
-  @GET
-  @Path("/{brokerId}/configs")
-  @Produces(Versions.JSON_API)
-  public void listBrokerConfigs(
-      @PathParam("clusterId") String clusterId, @PathParam("brokerId") String brokerId) {
-    throw new WebApplicationException(Status.NOT_IMPLEMENTED);
-  }
-
```

Comment:
We now have a separate resource for broker configuration, so removed from the `BrokerResource` class.